src/test/java/TestSqlUserService.java

```diff
@@ -0,0 +1,136 @@
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import week1.db.SqlUserServiceImpl;
+import week1.model.Address;
+import week1.model.User;
+
+import java.sql.*;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+
+public class TestSqlUserService {
+
+    static final String JDBC_DRIVER = "org.h2.Driver";
+    static final String DB_URL = "jdbc:h2:~/test";
+
+    static final String USER = "sa";
+    static final String PASS = "";
+
+    Connection conn = null;
+    Statement stmt = null;
+    SqlUserServiceImpl sqlUserService;
+
+
+    public TestSqlUserService() throws SQLException {
+    }
+
+    @Before
+    public void before() throws ClassNotFoundException, SQLException {
+
+        conn = DriverManager.getConnection(DB_URL, USER, PASS);
+        Class.forName("org.h2.Driver");
+
+        {
```

Comment:
try/catch is redundant here. new SqlUserServiceImpl(conn) is not throwing any exception.

Reply:
fixed